packages/web-react/src/types/stack.ts

```diff
@@ -0,0 +1,10 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import { ChildrenProps, StyleProps, TransferProps } from './shared';
+
+export interface SpiritStackProps<T extends ElementType = 'div'> extends ChildrenProps, StyleProps, TransferProps {
```

Comment:
In `elements.ts` there is a better base type `SpiritDivElementProps` prepared from @tomassychra that you can use instead of `StyleProps` and `JSXElementConstructor`.

Reply:
I already thought about it, but here it would be better to use the PolymorphicElementPropsWithoutRef similar to Button. Because here it is not only `div`, but `ul` is also used in the example. Then the `TransferProps` can be removed.

Reply:
Refactor those types using `PolymorphicElementPropsWithoutRef` but missing `IntrinsicAttributes`. Currently ignoring this error. Fixable in the future.